packages/tupaia-web/src/features/Map/MapOverlaySelector/DesktopMapOverlaySelector.tsx

```diff
@@ -149,45 +150,47 @@ export const DesktopMapOverlaySelector = ({
   const { data: mapOverlayData } = useMapOverlayReport();
 
   return (
-    <Wrapper>
-      <Header>
-        <Heading>Map Overlays</Heading>
-      </Header>
-      <Container>
-        <TitleWrapper>
-          <MapOverlaySelectorTitle />
-          <MapOverlayDatePicker />
-        </TitleWrapper>
-        {hasMapOverlays && (
-          <OverlayLibraryAccordion
-            expanded={overlayLibraryOpen}
-            onChange={toggleOverlayLibrary}
-            square
-          >
-            <OverlayLibraryHeader
-              expandIcon={<ArrowDropDown />}
-              aria-controls="overlay-library-content"
-              id="overlay-library-header"
+    <ErrorBoundary>
```

Comment:
If we have an error boundary in MapLegend.tsx then we shouldn't need this one as well